src/controllers/BaseController.ts

```diff
@@ -51,12 +51,12 @@ export abstract class BaseController<FORM> extends BaseAsyncHttpController {
     }
 
     protected prepareViewModelFromSession(session: Session): Record<string, any> {
-        const appeal: Appeal = session
+        const applicationData: ApplicationData = session
             .getExtraData()
-            .chain<Appeal>(data => Maybe.fromNullable(data[APPEALS_KEY]))
-            .orDefault({} as Appeal);
+            .chain<ApplicationData>(data => Maybe.fromNullable(data[APPEALS_KEY]))
```

Comment:
does `data[APPEALS_KEY]` return ApplicationData or the Appeals object?

Reply:
returns ApplicationData

Reply:
`APPEALS_KEY` can be renamed in the future but it would trigger to many unrelated changes if we have done it now.

Reply:
Actually I decided to rename it since it did not generate more additional changes.